lcs/agents/xcs/ClassifiersList.py

```diff
@@ -0,0 +1,149 @@
+import numpy as np
+import random
+import logging
+
+from lcs import TypedList, Perception
+from lcs.agents.xcs import Classifier, Condition, Configuration
+
+logger = logging.getLogger(__name__)
+
+
+class ClassifiersList(TypedList):
+    def __init__(self,
+                 cfg: Configuration,
+                 *args,
+                 oktypes=(Classifier,),
+                 ) -> None:
+        self.cfg = cfg
+        super().__init__(*args, oktypes=oktypes)
+
+    def insert_in_population(self, cl: Classifier):
+        for c in self:
```

Comment:
To jest newralgiczny moment. Na 100% musisz być pewien że metoda == dla klasy Classifier działa dobrze. Wg mnie klasyfikatory powinny być równe jeśli mają te same condition i action (nie klasy ale wartości). Reszta parametrów jest nieistotna.

Straciłem chyba tygodnie na debugowanie błędów tego rodzaju :)